src/components/new-request/search.vue

```diff
@@ -178,6 +178,28 @@
       </v-col>
     </v-row>
 
+    <!-- business lookup -->
+    <v-row no-gutters>
```

Comment:
I know that using business lookup is a work in progress, but could you make this conditional to CHG, AML, CNV and REH name requests only? That way it won't show for the other flows. Thanks.

Reply:
^^ to do in a future PR later today or tomorrow